app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt

```diff
@@ -56,7 +58,7 @@ class LoginFragment : Fragment() {
         savedInstanceState: Bundle?
     ): View? {
         rootView = inflater.inflate(R.layout.fragment_login, container, false)
-
+        CoordinatorLayout = rootView.findViewById(R.id.loginCoordinatorLayout)
```

Comment:
Don't use findViewById

Reply:
I have no idea why is this capital. Is this a class?

Reply:
> Don't use findViewById

Pls suggest alternate way